templates/lxd/index.html

```diff
@@ -0,0 +1,299 @@
+{% extends 'base_index.html' %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/1LJ6ajHYAmKdxSNDaD5kbriDod1t7RcmIwVJtevAeKFQ{% endblock meta_copydoc %}
+
+{% block title %}Canonical LXD{% endblock %}
+
+{% block meta_description %}Low-touch virtual infrastructure with LXD at any scale. Built on top of Linux containers, LXD offers a unified user experience in managing system containers, virtual machines and clusters. Easily configurable, flexible and secure way to run your workloads.{% endblock %}
+
+{% block body_class %}is-paper{% endblock body_class %}
+
+{% block content %}
+
+<section class="p-strip--whitesuru is-shallow u-no-padding--bottom">
+  <div class="p-section">
+    <div class="row--25-75">
+      <div class="col p-section">
+        <div class="p-section--shallow">
+          <h1 class="u-no-margin--bottom">Run system containers with LXD</h1>
+          <h2>Fast, dense, and secure container and VM management at any scale</h2>
+        </div>
+        <div class="p-section--shallow">
+          <p>LXD provides a unified user experience for managing system containers and virtual machines. For more demanding workloads, LXD can be set up in a cluster environment to run containers, VMs, or a combination of the two on a set of machines. LXD has direct hardware access, minimising overhead and matching the density and efficiency of containers.</p>
+        </div>
+        <hr class="u-hide--medium u-hide--small">
+        <a class="p-button--positive" href="/lxd/install">Install now</a>
+        </p>
+      </div>
+    </div>
+    <div class="row u-hide--medium u-hide--small">
+      <div class="col-9 col-start-large-4">
+        {{ image (
+        url="https://assets.ubuntu.com/v1/410648c5-Artboard%2070@2x.png",
+        alt="",
+        width="496",
+        height="375",
+        hi_def=True,
+        loading="auto"
+        ) | safe
+        }}
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width p-section--shallow">
+		<hr class="p-rule">
+		<h2>Why choose LXD?</h2>
```

Comment:
Weird indentation here, but not sure if it's just a GH thing

Reply:
that's a GH glitch indeed